tests/libnet/namespace.go

```diff
@@ -0,0 +1,114 @@
+package libnet
+
+import (
+	"context"
+	"encoding/json"
+
+	. "github.com/onsi/gomega"
+
```

Comment:
This import is fishy; helpers shouldn't use gomega / ginkgo.

Why would you need matchers on this file ? I see that you remove the import in the next commit; please squash that 'removal' in this one.

Reply:
There were forgotten `Expect` calls by mistake which is why the import remained.
I have [squashed](https://github.com/kubevirt/kubevirt/compare/2f31699ae8a76b1606c62c82c3f7fa3379eb52ac..622b1e4048c0f95f1c28026c45e26480e9f7ff97) the e2e test commit with the commit that refactored the namespace helpers - there's no need to have that commit separated